### DISCUSSION TITLE
Installing Stuck at boot logo

### DISCUSSION THREAD

**Discussion by igawa6** [stevedat/HP-Elitebook-830-G6-Hackintosh#12] [Q&A]
Hi mate, thanks for sharing the progress of your hackintosh.

I have the same Laptop but with only 8 gb of ram.

I'm new to this hackintosh thing. I read the dortania guide, and end up downloading Monterey and copying your EFI folder to the Flash Drive.

I finally make the opencore load but stuck at the first apple logo progress bar (half bar). Is it right to copy your EFI file or i need different EFI to first boot up

Sorry I'm newbie to hackintosh things. Thanksss

**Comment by stevedat** [comment]
Try my install EFI on big sur.
